snapshots/snapshot_reporter/src/snapshot_reporter.py

```diff
@@ -252,30 +253,38 @@ def get_recent_update_stats(session, *, hours):
                     }
                 ]
             }
-        },
+        }
     }
-    works_resp = pipeline_es_client.search(
-        index=works_index_name, body=indexed_time_query
-    )
-    images_resp = pipeline_es_client.search(
-        index=images_index_name, body=indexed_time_query
-    )
+    last_update_body = {
+        "size": 1,
+        "_source": ["debug.indexedTime"],
+        "sort": [{"debug.indexedTime": {"order": "desc"}}],
+    }
+    works_count = pipeline_es_client.count(
+        index=works_index_name, body=indexed_time_body
+    )["count"]
+    images_count = pipeline_es_client.count(
+        index=images_index_name, body=indexed_time_body
+    )["count"]
+
+    works_last_update = pipeline_es_client.search(
+        index=works_index_name, body=last_update_body
+    )["hits"]["hits"][0]["_source"]["debug"]["indexedTime"]
+    images_last_update = pipeline_es_client.search(
+        index=images_index_name, body=last_update_body
+    )["hits"]["hits"][0]["_source"]["debug"]["indexedTime"]
```

Comment:
This is the sort of place where I really miss either (1) JavaScript’s dot syntax for looking up properties or (2) the ability to do multi-line commands without breaking out backslash escapes – I find heavily nested dictionaries kinda ugly in Python.

(Nothing you can do about it, just musing!)